tests/conftest.py

```diff
@@ -384,7 +387,7 @@ def pytest_collection_modifyitems(config, items):
     for item in items:
         if "cloud_server" in item.fixturenames:
             cloud_type = item.callspec.params.get("cloud_type")
-            if cloud_type == "file":
+            if cloud_type not in cloud_types:
```

Comment:
Trying to be more defensive, so that we don't accidentally skip tests that we should not have.